src/react/components/StepNavigation.tsx

```diff
@@ -22,9 +23,13 @@ const StepNavigation: FC<Props> = (props): ReactElement => {
           <Button variant="contained" color="primary" disabled={props.disableBack} onClick={props.onPrev} tabIndex={3}>{props.backLabel}</Button>
         )}
       </Grid>
-      <Grid item xs={4} />
+      <Grid item xs={4}>
+        {!props.hideNext && props.centerNext && (
+          <Button variant="contained" color="primary" disabled={props.disableNext} onClick={props.onNext} tabIndex={2}>{props.nextLabel}</Button>
+        )}
+      </Grid>
       <Grid item xs={2}>
-        {!props.hideNext && (
+        {!props.hideNext  && !props.centerNext && (
```

Comment:
In cases where there is only one button and it should be centered, I think it would be better to not use the StepNavigation component, and just use a centered button instead.

Reply:
Good call.  Done!